README.md

```diff
@@ -786,7 +818,23 @@ vars:
     util_prefixes: ['util_']
 ```
 
+#### Performance Variables:
+
+| variable    | description | default     |
+| ----------- | ----------- | ----------- |
+| `chained_views_threshold` | threshold for unacceptable length of chain of views for `fct_chained_views_dependencies` | 4 |
+
+```yml
+# dbt_project.yml
+# set your chained views threshold to 8 instead of 5
```

Comment:
We talk about the default being 4 but then say "change from 5". I guess it is just a typo.

Reply:
typo! I'll fix it!